deploy/deploymentConfig.yaml

```diff
@@ -0,0 +1,37 @@
+---
+  kind: "DeploymentConfig"
+  apiVersion: "v1"
+  metadata:
+    name: "ose-eventforwarder"
+  spec:
+    template:
+      metadata:
+        labels:
+          name: "ose-eventforwarder"
+      spec:
+        containers:
+          -
+            name: "ose-eventforwarder"
+            image: >-
+              <your docker registry here>
+            env:
+            - name: SYSLOG_SERVER
+              valueFrom:
```

Comment:
See above commend. Use env variable directly?

Reply:
Same answer as above.  This just fits our deployment process, where we see the "deploymentConfig.yaml" being the application definition, and the "config map" entries being the variables that change from OpenShift cluster to OpenShift cluster.